include/ObjectManager.php

```diff
@@ -602,9 +602,7 @@ public function modifyNotifierMethod($method_name, $method_type,$new_method_name
 	
 	/* LILAC - List Hosts */
 	public function listHosts( $hostName = false, $hostTemplate = false ){
-		
-		return true;
-		
+		return $this->listNagiosObjects("hosts")["default"];
```

Comment:
I would add a defense on the "default" key, in the doubt that the returned array is empty.